### PR TITLE
Add some examples and improve older ones

### PR DESCRIPTION
**Pull request by meew0**
I haven't tested these, please review them for correctness and test them if necessary.

### CODE REVIEW

**Comment by Tvde1** [comment]
`<Client>.sendMessage()` when

File: examples/avatar.js [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+
+		// Check whether the user actually has an avatar.
 		if ( user.avatar ) {
+			// Construct the avatar URL from the user ID and the avatar ID.
 			var url = "https://discordapp.com/api/users/" + user.id + "/avatars/" + user.avatar + ".jpg";
+
+			// A user can be mentioned in a message by inserting the string obtained
+			// by user.mention() into the message.
+			// Note that simply writing "@user" will NOT work.
 			bot.sendMessage( message.channel, message.author.mention() + ", here's your avatar: " + url );
```
Comment (Jamelele) [inline examples/avatar.js]: Inconsistency between use of 'bot' and 'myBot' here

File: examples/pingpong.js [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
 
+// Add a listener to the "message" event, which triggers upon receiving
+// any message
 myBot.on( "message", function( message ) {
+	// message.content accesses the content of the message as a string.
+	// If it is equal to "ping", then the bot should respond with "pong".
 	if ( message.content === "ping" ) {
+		// Send a message ("pong") to the channel the message was sent in,
+		// which is accessed by message.channel.
 		this.sendMessage( message.channel, "pong" );
```
Comment (Jamelele) [inline examples/pingpong.js]: Inconsistent again, use 'this' or 'myBot'?

**Comment by Jamelele** [comment]
lmao what am I doing, just realised how old this PR is, it just popped up in my feed again because it was referenced

**Comment by hydrabolt** [comment]
@Jamelele this is brilliant it made my day 😆 👍

**Comment by Tvde1** [comment]
Same for me I was so confused when reading this.

**Comment by ItzSpectral** [comment]
stolica brate od 200 eura,znas ono

**Comment by DuckoDas** [comment]
I like fish

**Pull request merged**